### PR TITLE
Need to update css for RTL

### PR DESCRIPTION
**Pull request by nathanbegbie**

### CODE REVIEW

**Comment by nathanbegbie** [comment]
@SteveBarnett @miltontony review please

File: unicorecmsebola/templates/base.pt [hunk 1]
```diff
@@ -13,7 +14,7 @@
     <meta name="viewport" content="width=device-width; initial-scale=1.0; maximum-scale=1.0; user-scalable=0;" />
     <meta name="description" content=""/>
     <meta name="keywords" content="moms,maternal,baby,pregnancy,pregnant,expectant,mama,mother,pain,birth,help"/>
-    <link href="${request.static_url('cms:static/css/style.css')}" rel="stylesheet" type="text/css" />
+    <link href="${request.static_url('cms:static/css/style.css')}?v=20150129" rel="stylesheet" type="text/css" />
```
Comment (SteveBarnett) [inline unicorecmsebola/templates/base.pt]: @nathanbegbie But you didn't make any CSS changes?

**Comment by nathanbegbie** [comment]
@SteveBarnett Er, correct. I put that there because my browser had cached the css and I needed it to update, so that I could see what it currently looked like. It shall be removed!

**Comment by nathanbegbie** [comment]
@SteveBarnett Did I do it correctly this time?

**Comment by nathanbegbie** [comment]
@milton +1 please ?

**Comment by nathanbegbie** [comment]
@milton +1 ?

**Comment by miltontony** [comment]
:+1:

**Pull request merged**